dnf-behave-tests/features/search-alpha.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Sort search command output
+
+Background:
+  Given I use repository "dnf-ci-fedora"
+
+Scenario: sort alphanumerically
+  When I execute dnf with args "search --alpha fedora"
+  Then the exit code is 0
+   And stdout is
+   """
+   ======================== Name & Summary Matched: fedora ========================
+   fedora-release.noarch : Fedora release files
+   fedora-release.src : Fedora release files
+   =========================== Summary Matched: fedora ============================
+   basesystem.noarch : The skeleton package which defines a simple Fedora system
+   basesystem.src : The skeleton package which defines a simple Fedora system
+   """
```

Comment:
This doesn't seem like it necessarily tests the output is actually sorted...

In case we'd keep the `--alpha` option (see my https://github.com/rpm-software-management/dnf/pull/1610#issuecomment-606622177), the proper way to test it would be to call `search` without `--alpha` that would have a different order than when sorted.

In case we ditch the option, I guess this is fine, though it would be somewhat preferable to have at least two packages listed in one of the groups :slightly_smiling_face: